installation/scripts/generate-compass-installer-artifacts.sh

```diff
@@ -32,4 +33,20 @@ function generateArtifact() {
     rm -rf ${TMP_CR}
 }
 
+function copyKymaInstaller() {
+    release=$(<"${RESOURCES_DIR}"/KYMA_VERSION)
+
+if [[ $release == *PR-* ]] || [[ $release == *master* ]]; then
+    curl -L https://storage.googleapis.com/kyma-development-artifacts/${release}/kyma-installer-cluster.yaml -o kyma-installer.yaml
```

Comment:
We should also copy the `is-installed.sh` file and move it to `${ARTIFACTS_DIR}/is-kyma-installed.sh` here.